lib/spandx/php/packagist_gateway.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Spandx
+  module Php
+    class PackagistGateway
+      attr_reader :http, :source
+
+      def initialize(http: Spandx.http, source: 'https://repo.packagist.org')
+        @source = source
+        @http = http
+      end
+
+      def licenses_for(name, version)
+        response = http.get("#{source}/p/#{name}.json")
```

Comment:
@mokhan 

why do we need to make http request here ? `composer.lock` already provides that information

Reply:
I will fix this. Thx @eldemcan .